### PR TITLE
WR330605 settings page broken on moodle version lower than 3.1

### PR DESCRIPTION
**Pull request by gleimermora**

### CODE REVIEW

File: classes/local/store/object_client_base.php [hunk 1]
```diff
@@ -108,20 +108,39 @@ public function define_client_check($client) {
...
+    private function notification($message, $type = 'success') {
+        global $CFG, $OUTPUT;
+        $methodsmap = [
+            'success' => 'notify_success',
+            'warning' => 'notify_problem',
+            'error' => 'notify_problem',
+        ];
+        if (!empty($CFG->branch) && $CFG->branch < '31' && isset($methodsmap[$type])) {
+            $method = $methodsmap[$type];
+            $OUTPUT->$method($message);
```
Comment (brendanheywood) [inline classes/local/store/object_client_base.php]: why not just call

public function notification($message, $type = null) {

$OUTPUT->motification($message, $type); ?

File: classes/local/store/object_client_base.php [hunk 2]
```diff
@@ -108,20 +108,39 @@ public function define_client_check($client) {
         $SESSION->notifications = [];
```
Comment (brendanheywood) [inline classes/local/store/object_client_base.php]: I don't think you need this anymore, or the line 103 as well

File: classes/local/store/object_client_base.php (hunk 1 above)
Comment (gleimermora) [reply classes/local/store/object_client_base.php]: `$OUTPUT->$method` is deprecated on versions 3.1+ and it throws an exception.

**Comment by gleimermora** [comment]
new branch: https://github.com/catalyst/moodle-tool_objectfs/pull/237

**Pull request closed without merging**